extensions/positron-python/src/client/pythonEnvironments/nativeAPI.ts

```diff
@@ -478,6 +490,12 @@ class NativePythonEnvironments implements IDiscoveryAPI, Disposable {
         }
         const native = await this.finder.resolve(envPath);
         if (native) {
+            // --- Start Positron ---
+            if (native.executable && (await isUvEnvironment(native.executable))) {
+                traceInfo(`Found uv environment: ${native.executable}`);
+                native.kind = NativePythonEnvironmentKind.Uv;
```

Comment:
Since PET recognizes uv envs as venvs, here's where we patch them. Related: https://github.com/microsoft/python-environment-tools/issues/152#issuecomment-2543989037